src/cleed++/LEED/Crystal.hh

```
@@ -0,0 +1,116 @@
+    int getMirrorSymmetry() const;
+    const real *getAngleAlphaDegreesPtr() const;
+    vector<double> &getAngleAlphaDegrees() const;
+
+    vector<double> getSubstrateBasisVectors() const;
+    double getSubstrateUnitCellArea() const;
+    vector<double> getSuperstructureBasisVectors() const;
+    double getSuperstructureRelativeUnitCellArea() const;
+
+    size_t getNumberOfLayers() const;
+    const leed_layer_t *getLayersPtr() const;
+    vector<leed_layer_t> &getLayers() const;
```

Clang (14.0.6)

[warning] function 'getLayers' should be marked [[nodiscard]]

(modernize-use-nodiscard)